packages/analytics-js/src/state/slices/session.ts

```diff
@@ -4,14 +4,14 @@ import { ApiObject, SessionInfo } from '@rudderstack/analytics-js/state/types';
 import { Nullable } from '@rudderstack/analytics-js/types';
 
 export type SessionState = {
-  rl_user_id: Signal<Nullable<string> | undefined>;
-  rl_trait: Signal<Nullable<ApiObject> | undefined>;
-  rl_anonymous_id: Signal<string | undefined>;
-  rl_group_id: Signal<Nullable<string> | undefined>;
-  rl_group_trait: Signal<Nullable<ApiObject> | undefined>;
-  rl_page_init_referrer: Signal<string | undefined>;
-  rl_page_init_referring_domain: Signal<string | undefined>;
-  rl_session: Signal<SessionInfo>;
+  readonly rl_user_id: Signal<Nullable<string> | undefined>;
+  readonly rl_trait: Signal<Nullable<ApiObject> | undefined>;
+  readonly rl_anonymous_id: Signal<string | undefined>;
+  readonly rl_group_id: Signal<Nullable<string> | undefined>;
+  readonly rl_group_trait: Signal<Nullable<ApiObject> | undefined>;
+  readonly rl_page_init_referrer: Signal<string | undefined>;
+  readonly rl_page_init_referring_domain: Signal<string | undefined>;
+  readonly rl_session: Signal<SessionInfo>;
```

Comment:
Why you are making this readonly? we won't be able to update it once it is initialized.